test/ui-2/test/DropDownButtonStep.test.js

```diff
@@ -0,0 +1,20 @@
+const timeout = process.env.SLOWMO ? 30000 : 10000;
+const fs = require('fs');
+beforeAll(async () => {
+  path = fs.realpathSync('file://../examples/index.html');
+  await page.goto('file://' + path, {waitUntil: 'domcontentloaded'});
+});
+describe('add step dropdown', ()=>{
+  test('Click Button', async()=>{
+    await page.waitForSelector('.step');
+    const Length = await page.evaluate(() => document.querySelectorAll('.step').length);
+    await page.click('[data-value=\'brightness\']');
+
+    await page.waitForSelector('#add-step-btn');
+    page.click('#add-step-btn');
+    const LengthChanged = await page.evaluate(() => document.querySelectorAll('.step').length);
+    expect(Length).toBe(1);
+    expect(LengthChanged).toBe(2);
+
+  });
```

Comment:
I think you forgot to put the `timeout` here as the async function ends.

Reply:
Done @debck